mod/module_test.go

```diff
@@ -26,8 +26,9 @@ func TestConfigGitHubMode(t *testing.T) {
 }
 
 func TestConfigGoModulesMode(t *testing.T) {
-	fs := afero.NewMemMapFs()
+	fs := afero.NewOsFs()
```

Comment:
can this use `MemMapFs` instead?

Reply:
nah this runs `go mod init` which will create `go.mod` file in os filesystem